tests/integration/test_crunch_cube.py

```diff
@@ -1360,23 +1365,29 @@ def test_pets_x_pets_array_by_col(self):
     def test_pets_x_pets_array_by_row(self):
         cube = CrunchCube(FIXT_PETS_X_PETS_ARRAY)
         expected = np.array([
-            [0.44444444, 0.41176471],
-            [0., 1.],
-            [0.5, 0.47368421],
+            [0.55555556, 0.19444444],
+            [0., 0.55555556],
+            [0.44444444, 0.25],
```

Comment:
I'm not sure what's going on with these numbers, I'm not seeing anything like this. When I run proportions by row in R I get:

```
> prop.table(pets_array, 1)
, , pets_array = not selected

        pets_array
pets           cat       dog    wombat
  dog    0.1851852 0.0000000 0.1481481
  cat    0.0000000 0.1630435 0.1413043
  wombat 0.1929825 0.1754386 0.0000000

, , pets_array = selected

        pets_array
pets           cat       dog    wombat
  dog    0.1296296 0.3703704 0.1666667
  cat    0.3695652 0.1521739 0.1739130
  wombat 0.1403509 0.1578947 0.3333333
```

For reference, the same cube proportions by column are:
```
> prop.table(pets_array, 2)
, , pets_array = not selected

        pets_array
pets           cat    dog    wombat
  dog    0.2531646 0.0000 0.2285714
  cat    0.0000000 0.1875 0.1857143
  wombat 0.2784810 0.2500 0.0000000

, , pets_array = selected

        pets_array
pets           cat   dog    wombat
  dog    0.1772152 0.500 0.2571429
  cat    0.4303797 0.175 0.2285714
  wombat 0.2025316 0.225 0.5428571
```